backend/kesaseteli/applications/management/commands/send_vouchers_to_handlers.py

```diff
@@ -0,0 +1,48 @@
+from django.core.management.base import BaseCommand
+
+from applications.enums import YouthApplicationStatus
+from applications.models import YouthApplication
+
+
+class Command(BaseCommand):
+    help = "Send accepted youth summer vouchers to handler email"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="Dry run, don't really send anything",
+        )
+        parser.add_argument(
+            "--id", type=str, help="UUID of a single youth application to send"
+        )
+
+    def handle(self, dry_run, id, *args, **options):
+        apps = YouthApplication.objects.filter(status=YouthApplicationStatus.ACCEPTED)
+        if id:
+            # Only a single voucher is sent
+            apps = apps.filter(id=id)
+
+        for app in apps:
+            if app.has_youth_summer_voucher:
+                if dry_run:
+                    ok = True
+                else:
+                    ok = app.youth_summer_voucher.send_youth_summer_voucher_email(
+                        language=app.language, send_to_youth=False, send_to_handler=True
```

Comment:
All vouchers that has been accepted are sent to handler only but not resent to youth. Seems to be correct logic to me.